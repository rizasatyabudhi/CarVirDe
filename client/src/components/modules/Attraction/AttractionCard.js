import React, { Component } from "react";

export default class AttractionCard extends Component {
  render() {
    return (
      <div class="col s12">
        <div class="card horizontal hoverable">
          <div class="card-image large">
            <img
              src={this.props.foto}
              style={{ width: "230px", height: "217px" }}
            />
          </div>
          <div class="card-stacked">
            <div class="card-content">
              <h5 style={{ marginBottom: "20px" }}>{this.props.nama}</h5>
              <div className="row">
                <div className="col s3">
                  <ul>
                    <li>
                      <i className="material-icons">location_on</i>
                      {this.props.alamat}
                    </li>
                    <li>
                      <i className="material-icons">attach_money</i>
                      <span>
                        Rp
                        {this.props.harga}
                        /Pax
                      </span>
                    </li>
                  </ul>
                </div>
                <div className="col s9 paragraph" style={{ marginTop: "23px" }}>
                  {this.props.deskripsi}
                </div>
              </div>
            </div>

            <div class="card-action">
              <button
                class="btn waves-effect waves-light"
                type="submit"
                name="action"
                onClick={this.props.onClick}
              >
                Add to Cart
                <i class="material-icons right">add_shopping_cart</i>
              </button>
            </div>
          </div>
        </div>
      </div>
    );
  }
}
