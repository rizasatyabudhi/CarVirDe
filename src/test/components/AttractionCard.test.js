import React from "react";
import { shallow } from "enzyme";
import AttractionCard from "../../components/modules/Attraction/AttractionCard";

test("should render AttractionCard", () => {
  const wrapper = shallow(<AttractionCard />);
  expect(wrapper).toMatchSnapshot();
});
