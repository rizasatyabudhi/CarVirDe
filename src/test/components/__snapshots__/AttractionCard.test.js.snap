// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AttractionCard 1`] = `
<div
  class="col s12"
>
  <div
    class="card horizontal hoverable"
  >
    <div
      class="card-image large"
    >
      <img
        style={
          Object {
            "height": "217px",
            "width": "230px",
          }
        }
      />
    </div>
    <div
      class="card-stacked"
    >
      <div
        class="card-content"
      >
        <h5
          style={
            Object {
              "marginBottom": "20px",
            }
          }
        />
        <div
          className="row"
        >
          <div
            className="col s4"
          >
            <ul>
              <li>
                <i
                  className="material-icons"
                >
                  location_on
                </i>
              </li>
              <li>
                <i
                  className="material-icons"
                >
                  attach_money
                </i>
                <span>
                  Rp Â 
                  /Pax
                </span>
              </li>
            </ul>
          </div>
          <div
            className="col s8 paragraph"
            style={
              Object {
                "marginTop": "23px",
              }
            }
          />
        </div>
      </div>
      <div
        class="card-action"
      >
        <button
          class="btn waves-effect waves-light"
          name="action"
          type="submit"
        >
          Add to Cart
          <i
            class="material-icons right"
          >
            add_shopping_cart
          </i>
        </button>
      </div>
    </div>
  </div>
</div>
`;
